content/articles/effects-of-feature-scaling-on-a-machine-learning-model/index.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /effects-of-feature-scaling-on-a-machine-learning-model/
+title: Effects of Feature scaling on a machine learning model
+description: This article will discuss the effects of feature scaling on a machine learning model.
+author: samuel-mwangi
+date: 2021-12-17T00:00:00-19:30
+topics: [Machine learning]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/effects-of-feature-scaling-on-a-machine-learning-model/hero.jpg
+    alt: Effects of Feature Scaling Hero Image
+---﻿
+### Why is feature scaling important
+Datasets that we use for training models in machine learning have unpredictable values that might end up varying from each other on a broad scale. Numerical values might have big differences amongst themselves especially when they represent different scales, and this might make it difficult to compare them ie: kg, Litres, Millimeters, Miles, Pixels, etc.
+
+Feature scaling is introduced to solve this challenge. It adjusts the numbers in a manner that makes it easy to compare the values that are out of each other's scope. This helps increase the accuracy of the models especially those using algorithms that are sensitive to feature scaling ie: Gradient Descend and Distance-based algorithms.
+
+There are two techniques of scaling features:
+1. **Normalization** - the values are re-scaled to range between zero and one.
+2. **Standardisation** - the values are re-scaled to center around the mean with units of standard deviation.
+
+Choosing which to use on your dataset is subjective to your dataset, the machine learning algorithm, and the type of problem that you might be trying to solve.
+
+ In this approach, we will learn how to implement each. We will first build a prediction model without feature scaling, one with standardized features, and lastly one with Normalized features. We will also use the same dataset for the three to be able to compare how our dataset affects the choice of the technique we use.
+
+### Table of contents
+1.[Prerequisites](#prerequisites)
+
+2.[Importing the libraries](#importing-the-libraries)
+
+3.[The initial model](#1the-initial-model)
+
+4.[Using Standard Scaling](#2using-standard-scaling)
+
+5.[Using Normalization](#3using-normalization)
+
+6.[Effects of the feature scaling](#effects-of-the-feature-scaling)
+
+7.[Choosing between the two scaling techniques](#choosing-between-the-two-scaling-techniques)
+
+8.[Exceptions](#exceptions)
+
+9.[Conclusion](#conclusion)
```

Comment:
Use a dash instead of numbers. In addition, please remove the white space, it is unnecessary.

Reply:
Do you mean the lines between ?

Reply:
Yes. Those ones